integration-tests/weather/src/test/java/org/apache/camel/quarkus/component/weather/it/WeatherTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.weather.it;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.Matchers.containsString;
+
+@QuarkusTest
+class WeatherTest {
+
+    @Test
+    public void loadComponentWeather() {
+        /* A simple autogenerated test */
+        RestAssured.get("/weather/load/component/weather")
+                .then()
+                .statusCode(200);
+    }
+
+    @Test
+    public void loadByLocationName() {
+        RestAssured.given()
+                .get("/weather/location/London,uk")
+                .then()
+                .statusCode(200)
+                .body(containsString("name\":\"London\""), containsString("\"country\":\"GB\""), containsString("weather"),
```

Comment:
BTW, restassured allows for matching JSON payload attribs against Matchers https://github.com/rest-assured/rest-assured/wiki/Usage#example-1---json It would perhaps suit here too.

Reply:
done!! Thanks!! I've changed that.